benchmarks/python/test_rope.py

```diff
@@ -176,3 +62,80 @@ def test_rope_benchmark(
 
     if not disable_benchmarking:
         run_benchmark(benchmark, fd.execute, inputs)
+
+
+@pytest.mark.parametrize(
+    "rope_variation",
```

Comment:
```suggestion
    "variation",
```
The whole file is about rope and the benchmark name has `test_rope` in it.